.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# https://help.github.com/en/articles/about-code-owners#codeowners-syntax
+# Site Reliability Engineering (SRE) Reviewers
+*                                                  @bodyshopbidsdotcom/sre
+
+# SRE Owners
+/.github/workflows/pre-release.yml      @bodyshopbidsdotcom/sre
+/action.yml                             @bodyshopbidsdotcom/sre
```

Comment:
Does this need a `/` at the beginning?

Reply:
I was using VBE codeowners file as example https://github.com/bodyshopbidsdotcom/tinker/blob/main/.github/CODEOWNERS. 
I looked at out tinker repo and this is what we have [there](https://github.com/bodyshopbidsdotcom/tinker/blob/main/.github/CODEOWNERS).  I've updated it to this. 

```
# https://help.github.com/en/articles/about-code-owners#codeowners-syntax
* @bodyshopbidsdotcom/sre
```